### DISCUSSION TITLE
Palette Toggling and "Selected" Animations on Select Screen

### DISCUSSION THREAD

**Discussion by VinVulpis** [ikemen-engine/Ikemen-GO#1590] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Pretty minor and unsure if it's been suggested yet, but a nice presentation request I have is a way to preview palettes on the character select screen (I think this has been mentioned here but I've started tinkering with my own screenpack and figured I'd mention it).

Another feature that would be neat is if for team modes, if selected character animations stay on  screen, but change to a specified animation when selected (like a victory pose, taunt, or whatever). 

### Describe the solution you'd like

 I see that animations can be swapped for big portraits. My own ideas for this would be something like: certain buttons can be designated to flipping through palettes (like C and Z or D and W) and certain buttons be for selecting (Like A and X).

Maybe this can be implemented by a on/off switch in the screenpack for classic mugen palette selection, or a more modern form of it like many fighting games have now. This could make more room for even more palettes to be selectable, should people want more than 12.

For the team selections, maybe some kind of screenpack option like "p1.face.selected.anim=" to change the sprite or animation when the character is selected, then layer the next characters in with a "p1.face.spacing=" option. (So that they can stand together like they do on the VS screen) Maybe have an internal check for anim/sprite and just keep a default anim/sprite if the called anim/sprite don't exist.

### Additional context

I like options for presentation. Ways to make the select/VS screens more interesting would be great.

### Search terms

palette, select screen

**Comment by LuigiMario79** [comment, marked as answer]
Select animation is already implemented. Just paste this in your system.def under [Select Info]:
pX.face.done.anim = [selected animation number]

**Reply by VinVulpis** [reply]
Oh, well how about that, it works! Now if we can just get the whole team to stay on screen, that'd be awesome. 
Thanks for telling me this.
